app/services/core_data_connector/import/instances.rb

```diff
@@ -0,0 +1,116 @@
+module CoreDataConnector
+  module Import
+    class Instances < Base
+      def cleanup
+        super
+
+        execute <<-SQL.squish
+          WITH update_instances AS (
+            UPDATE
+            core_data_connector_instances
+              SET z_instance_id = NULL
+          )
+
+          UPDATE
+          core_data_connector_names
+            SET z_source_id = NULL,
+                z_source_type = NULL
+        SQL
+      end
+
+      def load
+        super
+
+        execute <<-SQL.squish
+          UPDATE core_data_connector_instances instances
+             SET z_instance_id = z_instances.id,
+                 user_defined = z_instances.user_defined,
+                 updated_at = current_timestamp
+            FROM #{table_name} z_instances
+           WHERE z_instances.instance_id = instances.id
+        SQL
+
+        execute <<-SQL.squish
+        WITH
+
+        insert_instances AS (
+          INSERT INTO core_data_connector_instances (
```

Comment:
For consistency, could we match the formatting of the SQL in the other import services:
- New line after the CTE name line (i.e. `insert_instances as ( \n`)
- Right  align the first keyword in each line except for `RETURNING` (`UPDATE`, `SET`, `FROM`, `WHERE`, etc)
- New line after the parenthesis closing the CTE table